src/celengine/nebula.cpp

```diff
@@ -62,6 +87,12 @@ Nebula::getObjType() const
 bool
 Nebula::load(const util::AssociativeArray* params, const fs::path& resPath, std::string_view name)
 {
+    
+    if (const auto* typeName = params->getString("Type"); typeName == nullptr)
+        setType({});
```

Comment:
I think we can just leave the type alone if it is not specified, since the field initializer already defaults it to `NotDefined`

Reply:
Fixed